Packs/GoogleSafeBrowsing/Integrations/GoogleSafeBrowsingV2/GoogleSafeBrowsingV2.yml

```diff
@@ -0,0 +1,113 @@
+- defaultvalue: https://safebrowsing.googleapis.com/v4/threatMatches:find
+  display: Your server URL
+  name: url
+  required: true
+  type: 0
+- display: The Client ID
+  hidden: false
+  name: client_id
+  required: false
+  type: 0
+- defaultvalue: '1.0'
+  display: The Client Version
+  hidden: false
+  name: client_version
+  required: false
+  type: 0
+- additionalinfo: The API Key to use for connection
+  display: API Key
+  name: api_key
+  required: true
+  type: 4
+- additionalinfo: Reliability of the source providing the intelligence data.
+  defaultvalue: B - Usually reliable
+  display: Source Reliability
+  name: integrationReliability
+  options:
+  - A+ - 3rd party enrichment
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: Search Safe Browsing, The Safe Browsing APIs (v4) let your client applications check URLs against Google's constantly updated lists of unsafe web resources.
+display: Google Safe Browsing v2
+name: Google Safe Browsing v2
+script:
+  commands:
+  - arguments:
+    - default: true
+      description: url
+      isArray: false
+      name: url
+      required: true
+      secret: false
+    deprecated: false
+    description: Searches for the specified URL on lists.
+    execution: false
+    name: url
+    outputs:
+    - contextPath: DBotScore.Indicator
+      description: The Indicator
+      type: string
+    - contextPath: DBotScore.Type
+      description: The Indicator Type
+      type: string
+    - contextPath: DBotScore.Vendor
+      description: The DBot score vendor
+      type: string
+    - contextPath: DBotScore.Score
+      description: The DBot score
+      type: number
+    - contextPath: DBotScore.Reliability
+      description: Reliability of the source providing the intelligence data.
+      type: String
+    - contextPath: URL.Malicious.Vendor
+      description: For malicious URLs, the vendor that made the decision
+      type: string
+    - contextPath: URL.Malicious.Description
+      description: For malicious URLs, the reason that the vendor made the decision
+      type: string
+    - contextPath: URL.Data
+      description: URL name
+      type: string
+    - contextPath: GoogleSafeBrowsingV2.URL.threatType
+      description: The URL threat type.
+      type: string
+    - contextPath: GoogleSafeBrowsingV2.URL.platformType
+      description: The URL platform type.
+      type: string
+    - contextPath: GoogleSafeBrowsingV2.URL.cacheDuration
+      description: The URL cache duration time.
+      type: string
+    - contextPath: GoogleSafeBrowsingV2.URL.threatEntryType
+      description: The URL threat entry type.
+      type: string
```

Comment:
please fix the contextpath in here as well and update the readme

Reply:
done